x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/signals/open_close_signals_route.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Hapi from 'hapi';
+import { DETECTION_ENGINE_SIGNALS_STATUS_URL } from '../../../../../common/constants';
+import { SignalsRequest } from '../../alerts/types';
+import { setSignalsStatusSchema } from '../schemas';
+import { ServerFacade } from '../../../../types';
+import { transformError } from '../utils';
+
+export const setSignalsStatusRouteDef: Hapi.ServerRoute = {
+  method: 'POST',
+  path: DETECTION_ENGINE_SIGNALS_STATUS_URL,
+  options: {
+    tags: ['access:siem'],
+    validate: {
+      options: {
+        abortEarly: false,
+      },
+      payload: setSignalsStatusSchema,
+    },
+  },
+  async handler(request: SignalsRequest, headers) {
+    const { signal_ids: signalIds, query, status } = request.payload;
+    const { callWithRequest } = request.server.plugins.elasticsearch.getCluster('data');
+    if (!callWithRequest) {
+      return headers.response().code(404);
+    }
+    if (status) {
+      let queryObject;
+      if (signalIds) {
+        queryObject = { ids: { values: signalIds } };
+      }
+      if (query) {
+        queryObject = JSON.parse(query);
+      }
+      try {
+        return callWithRequest(request, 'updateByQuery', {
+          index: '.siem-signals-devin-hurley',
```

Comment:
You might have to have this index hard coded like this until my PR goes through and you update yours. I would add at least a TODO above line `42` for now to remind yourself until mine goes through and you can see how to grab the space id and the index from the utility I provide.

Reply:
Ok my PR is through now, you can use the `getIndex` like in my code to get which index it is going to use.

Reply:
Thanks I just pushed this commit https://github.com/elastic/kibana/pull/52356/commits/a1b33df9936a4a8057a04835c7d7f091d9e9ce2e to add it 👍